src/lib/actions.ts

```
@@ -0,0 +1,40 @@
+  name: true,
+  email: true,
+})
+
+export async function firstLogin(
+  prevState: z.ZodError<z.infer<typeof FirstLoginSchema>> | void,
+  formData: FormData,
+) {
+  const session = await auth()
+  const data: { [key: string]: string } = {}
+  formData.forEach((value, key) => (data[key] = value.toString()))
+  data.email = session!.user!.email!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)